src/store/store.types.ts

```diff
@@ -0,0 +1,5 @@
+enum ESliceNames {
```

Comment:
Just suggestion, feel free to ignore it.
But I think there is no need in this enum, just write 'combineReducers({ categories: categoriesReducer })` and then in selector `state.categories.categories`. Typescript should check that there is no typos